runelite-client/src/main/resources/net/runelite/client/plugins/gpu/comp_common.glsl

```diff
@@ -38,6 +38,8 @@
    ivec2 sinCosTable[2048];
  };
 
+uniform mat4 projectionMatrix;
```

Comment:
The projection matrix in the compute shader is uninitialized so this completely breaks all of the face sorting.

![image](https://user-images.githubusercontent.com/309739/93917900-966c3c80-fcd9-11ea-8f4b-452496b29f07.png)

Reply:
I might be blind... but what exactly is wrong in that pic? Anyway, I fixed initializing the projection matrix.